easybuild/easyconfigs/l/LHAPDF/LHAPDF-6.5.3-gompi-2022a.eb

```
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>
+
+easyblock = 'ConfigureMake'
+
+name = 'LHAPDF'
+version = '6.5.3'
+
+homepage = 'http://lhapdf.hepforge.org/'
+description = """
+Les Houches Parton Density Function unified library
+
+First you need to adjust by `export LHAPDF_DATA_PATH=/local/.../path/` to your.
+Then you can run `lhapdf update` followed by `lhapdf install your_pdf`.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2022a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://www.hepforge.org/archive/lhapdf/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['57435cd695e297065d53e69bd29090765c934936b6a975ff8c559766f2230359']
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('Boost', '1.79.0'),
+]
+
+configopts = "--disable-static --enable-python"
+
+sanity_check_paths = {
+    'files': ["bin/lhapdf", "bin/lhapdf-config", "lib/libLHAPDF.%s" % SHLIB_EXT],
+    'dirs': ["share/LHAPDF"],
+} 
```

Sorry, missed another style issue (trailing whitespace):
```suggestion
}
```

---

Thanks. I am updating my own CI as well for better PRs in the future (mainly high energy physics,
https://gitlab.com/APN-Pucky/easyconfig-hep-forge/-/tree/master/)